lib/widgets/compose_box.dart

```diff
@@ -972,18 +965,16 @@ class _ComposeBoxContainer extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ColorScheme colorScheme = Theme.of(context).colorScheme;
+    final designVariables = DesignVariables.of(context);
 
     // TODO(design): Maybe put a max width on the compose box, like we do on
     //   the message list itself
-    return SizedBox(width: double.infinity,
+    return Container(width: double.infinity,
+      decoration: BoxDecoration(
+        border: Border(top: BorderSide(color: designVariables.borderBar))),
       child: Material(
-        color: colorScheme.surfaceContainerHighest,
-        child: SafeArea(
-          minimum: const EdgeInsets.fromLTRB(8, 0, 8, 8),
-          child: Padding(
-            padding: const EdgeInsets.only(top: 8.0),
-            child: child))));
+        color: designVariables.composeBoxBg,
+        child: SafeArea(child: child)));
```

Comment:
Could be a followup, but there's something in the current logic that would be nice not to regress on: when the device has left and right insets (as on an iPhone in landscape mode), we don't add more left and right padding on top of the padding that consumes the inset.

Compare:

| Current (branch tip) | Better (less wasted space) |
| --- | --- |
| <img width="1246" alt="image" src="https://github.com/user-attachments/assets/275237ed-facf-4c70-9bf1-84a811d16f52"> | <img width="1246" alt="image" src="https://github.com/user-attachments/assets/0c63450f-7a7b-485e-969d-34092189e2f2"> |

Here's the `git diff -b` to get that "better" version atop your branch:

```diff
$ git diff -b
diff --git lib/widgets/compose_box.dart lib/widgets/compose_box.dart
index f58b865d5..5e9beb87a 100644
--- lib/widgets/compose_box.dart
+++ lib/widgets/compose_box.dart
@@ -1031,7 +1031,8 @@ class _ComposeBoxContainer extends StatelessWidget {
         border: Border(top: BorderSide(color: designVariables.borderBar))),
       child: Material(
         color: designVariables.composeBoxBg,
-        child: SafeArea(child: child)));
+        child: SafeArea(left: false, right: false,
+          child: child)));
   }
 }
 
@@ -1076,16 +1077,17 @@ class _ComposeBoxLayout extends StatelessWidget {
 
     return _ComposeBoxContainer(
       child: Column(children: [
-        Padding(
-          padding: const EdgeInsets.symmetric(horizontal: 16),
+        SafeArea(
+          minimum: const EdgeInsets.symmetric(horizontal: 16),
           child: Theme(
             data: inputThemeData,
             child: Column(children: [
               if (topicInput != null) topicInput!,
               contentInput,
             ]))),
-        Container(
-          padding: const EdgeInsets.symmetric(horizontal: 8),
+        SafeArea(
+          minimum: const EdgeInsets.symmetric(horizontal: 8),
+          child: SizedBox(
             height: _composeButtonsRowHeight,
             child: IconButtonTheme(
               data: iconButtonThemeData,
@@ -1105,6 +1107,7 @@ class _ComposeBoxLayout extends StatelessWidget {
                   ]),
                   sendButton,
                 ]))),
+        ),
       ]));
   }
 }
```